### PR TITLE
[ART-3781] [ART-3783] Improve check-bugs job

### PR DESCRIPTION
**Pull request by locriandev**

### CODE REVIEW

File: jobs/build/check-bugs/Jenkinsfile [hunk 1]
```diff
@@ -1,8 +1,21 @@
 #!/usr/bin/env groovy
 node {
+    currentBuild.displayName = params.BUILD_VERSION
+
+    // Skip check for 3.11
+    if (params.BUILD_VERSION == "3.11") {
```
Comment (joepvd) [inline jobs/build/check-bugs/Jenkinsfile:6]: Think the skip for 3.11 should only happen for the regression check. We do want to have the check for bugs with the `blocker+` flag set.

**Review by joepvd — COMMENTED** [review]
Nice! One nit, then we can merge

File: jobs/build/check-bugs/Jenkinsfile (hunk 1 above)
Comment (locriandev) [reply jobs/build/check-bugs/Jenkinsfile]: fixed

**Comment by joepvd** [comment]
/lgtm

:shipit:

**Pull request merged**